docker-compose.yml

```diff
@@ -70,6 +71,7 @@ services:
       APP_PORT: 8080
       APP_URL: http://app:8080
       CELERY_LOG_FORMAT: human_readable
+      SIGNER_KEY_PASSWORD: 'pass'
```

Comment:
Minor comment here. Code analysis will complaint about hard-coding passwords in the Docker file. If not know, these secrets should be passed as environment variables.